### DISCUSSION TITLE
404 error GitHub Pages with correct README.md

### DISCUSSION THREAD

**Discussion by meteoritt** [community/community#22235] [Other Feature Feedback, Questions, & Ideas]
<p>Repository: <a href="https://github.com/CSRedRat/csredrat.github.io" rel="noopener nofollow ugc">https://github.com/CSRedRat/csredrat.github.io</a><br>
GitHub Pages: csredrat.gq</p>
<p>I publish only README.md, without index.md/index.html. On previous commit all work, but now I have 404 error. Why? README.md correct and displayed at repository browse. How fix this? Problem was appeared after I sync repo with GitBook: wiki.csredrat.gq</p>

**Comment by darkRaspberry** [comment]
<p>Just remove the <code>_config.yml</code> file from repo and it will work.<br>
When you added <code>_config.yml</code> file then GitHub pages think you are using a website with jekyll. Now you have to add a <code>index.markdown</code> or just <code>index.md</code> to show your home page. If you  add a <code>index.markdown</code> file it will be work.</p>
<p>Now you have to read offical docs to use jekyll <a href="https://jekyllrb.com/" rel="noopener nofollow ugc">https://jekyllrb.com/</a>  .</p>

**Comment by meteoritt** [comment]
<p>Pages broken before I add _config.yml, on previous commit site work, but after modify README.md in gitbook I have 404 error. In _config.yml set theme for md-files</p>

**Comment by meteoritt** [comment]
<p>I’m revert GitBook commit, but error don’t fix.</p>

**Comment by meteoritt** [comment]
<p>I rollback to working commit and site will work. Where I have problem with GitBook commit?</p>

**Comment by meteoritt** [comment, marked as answer]
<p>Now all work. May be because I delete draft in GitBook or old branch.</p>

**Comment by darkRaspberry** [comment]
<p>I guess you have problem with config. If you are using jekyll please refer to docs.</p>

**Comment by meteoritt** [comment]
<p>Config generated automatic by GitLab and correct. I don’t use Jekyll manual, I only add Markdown files, Jekyll themes applied from repository setting. Now all work fine, I rollback and remove old branch, remove about page info and draft in GitBook.</p>
